packages/manager/modules/pci/src/projects/project/storages/databases/components/allowed-ip/allowed-ip.html

```diff
@@ -0,0 +1,40 @@
+<form name="allowedIp" data-ng-submit="$ctrl.processIp()">
```

Comment:
Is there a reason to choose form validation here instead of `oui-modal` primary-action ?

Reply:
No reason. primary action is now being used :)
thanks